plugins/bitrise/src/components/BitriseBuildDetailsDialog/BitriseBuildDetailsDialog.test.tsx

```diff
@@ -48,7 +49,7 @@ describe('BitriseArtifactsComponent', () => {
 
     expect(rendered.queryByText('VISIBLE')).not.toBeInTheDocument();
 
-    btn.click();
+    await act(() => userEvent.click(btn));
```

Comment:
It should but wasn't. Turns out it was actually a quite serious and completely hidden issue. The way it works is that the `@testing-library` packages talk to each other under the hood and patch behavior, so when you use `user-event` together with `react` then it'll patch `dom` to automatically `act` on `user-events`. That was all broken because I only bumped to RTL 13, which caused a duplicate installation of `dom` under the hood, which in turn cause the patching to fail.

I bumped all the way up to RTL 14 now and that removed fixed things and I could remove these explicit `act`s. Yet another thing to note in the React 18 migration guide 😁